react/components/helpers.ts

```diff
@@ -0,0 +1,61 @@
+export const slugify = (text: string) => {
```

Comment:
:pick: I would move this file closer to where it's used (`react/component/admin`), and rename it to `utils.ts` to follow the project's naming convention.

Reply:
I wasn't sure where to put this file since it's being used for both pages form. I'll accept the advice.

Reply:
As a rule of thumb, I usually try to keep files as close as possible to the place they're being used; in that case, it's imported from `react/components/admin/institutional/Form/Form.tsx`, `react/components/admin/institutional/Form/utils.tsx` and `react/components/admin/pages/Form/index.tsx`, so the closest common point would be `react/components/admin`, hence my suggestion 🙂